django/core/management/commands/shell.py

```diff
@@ -47,18 +48,21 @@ def add_arguments(self, parser):
     def ipython(self, options):
         from IPython import start_ipython
 
-        start_ipython(argv=[])
+        start_ipython(
+            argv=[],
+            user_ns=(self.get_and_report_namespace(options["verbosity"])),
```

Comment:
Drop unnecessary parentheses

```suggestion
            user_ns=self.get_and_report_namespace(options["verbosity"]),
```